### ISSUE TITLE
chore: keyboards that target touch but are missing .js in packages

### ISSUE THREAD

**Issue by mcdurdin** [keymanapp/keyboards#2303]
```
balochi_scientific.kps - warn KM0401A: Keyboard balochi_scientific targets touch devices but corresponding balochi_scientific.js file is not in the package.
gff_amh_7.kps - warn KM0401A: Keyboard gff_amh_7 targets touch devices but corresponding gff_amh_7.js file is not in the package.
gff_amh_powerpack_7.kps - warn KM0401A: Keyboard gff_amh_7 targets touch devices but corresponding gff_amh_7.js file is not in the package.
gff_gurage_and_amharic.kps - warn KM0401A: Keyboard gff_amharic targets touch devices but corresponding gff_amharic.js file is not in the package.
gff_amh_powerpack_7.kps - warn KM0401A: Keyboard gff_ethiopic_7 targets touch devices but corresponding gff_ethiopic_7.js file is not in the package.
gff_ethiopic_7.kps - warn KM0401A: Keyboard gff_ethiopic_7 targets touch devices but corresponding gff_ethiopic_7.js file is not in the package.
gff_gurage_and_amharic.kps - warn KM0401A: Keyboard gff_gurage targets touch devices but corresponding gff_gurage.js file is not in the package.
lao_phonetic.kps - warn KM0401A: Keyboard lao_phonetic targets touch devices but corresponding lao_phonetic.js file is not in the package.
```

**Comment by DavidLRowe** [comment]
#2308 fixes all but:
```
gff_amh_7.kps
gff_amh_powerpack_7.kps
gff_ethiopic_7.kps
```
which are deprecated.

**Comment by mcdurdin** [comment]
> How do you want that handled? Or maybe you should not be testing deprecated keyboards?

So currently the keyboards are listed as supporting mobile on keyman.com, although they don't appear in default searches as they are deprecated.

In order to keep things neat and tidy, it'd be good to tidy up the discrepancy. It's not going to impact end users much but it helps to reduce noise in the builds and avoid confusion in the future.

Either we remove the 'any' from the &targets store, or add the .js to the .kps. If we remove the 'any' from the &targets store, we don't need to bump the version -- it will mean that the keyboards won't be listed as supporting mobile any more because the .keyboard_info is always updated with every build -- at least for the database, if not for downloads.keyman.com.

But if we add .js to the .kps, we need to bump the version, or the updated keyboard won't be published, and the data on keyman.com will still not line up with the downloads available.

**Comment by mcdurdin** [comment]
Fixed in #2308 and #2310
